features/errors.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Errors handling
+
+  Scenario: Reports an error when the given API key is incorrect
+    Given no project exist on localeapp.com with API key "MYAPIKEY"
+    When I run `localeapp -k MYAPIKEY add foo en:bar`
+    Then the exit status must be 70
+    And the output must match /error.+404/i
+
+  # FIXME: we should have only *one* test for this, result should not differ
+  # depending on the command. If we don't fix/refactor this, then it means we
+  # should test/handle and maybe fix error handling for *all* commands in their
+  # respective feature file.
+  # Scenario: Reports an error when the given API key is incorrect
+  #   Given no project exist on localeapp.com with API key "MYAPIKEY"
+  #   When I run `localeapp -k MYAPIKEY pull`
+  #   Then the exit status must be 70
+  #   And the output must match /error.+404/i
```

Comment:
I thought error handling would be more centralized than it is, given
we have many "base" classes like `Localeapp::CLI::Command`,
`Localeapp::ApiCall` etc…

But currently we don't really have such thing, either it's handled
directly in the command, or in the objects to which the command sends
message, or as a fallback it's GLI error handler (when exceptions are
raised).

Either we try to improve this, and we could test (UAT) generic errors
with whatever command is appropriate, either we limit changes to
improve error handling, but we need to test all errors we want to
handle on *all* commands, and potentially make multiple fixes in
commands, or classes those command rely on.